### PR TITLE
Issue #4 : Allow debug

### PR DESCRIPTION
**Pull request by tiaped**
Hi I gave it a try :)

It seems to work. But I have no idea as how to test it.

Sorry if I messed some things with the pull request, I'm not used to it.

Awaiting feedback.

### CODE REVIEW

File: src/main/kotlin/io/vertx/gradle/VertxPlugin.kt [hunk 1]
```diff
@@ -177,5 +179,48 @@ class VertxPlugin : Plugin<Project> {
...
+    }
+
+    logger.debug("The vertxRun task has been configured")
+  }
+
+  private fun computeDebugOptions(project: Project): List<String> {
+    val vertxExtension = project.vertxExtension()
+    val debugger = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=" +
+      (if (vertxExtension.debugSuspend) "y" else "n") + ",address=" + vertxExtension.debugPort
+    val disableEventLoopchecker = "-Dvertx.options.maxEventLoopExecuteTime=" + java.lang.Long.MAX_VALUE
```
Comment (jponge) [inline src/main/kotlin/io/vertx/gradle/VertxPlugin.kt]: Good catch.

This should be documented BTW, like adding a sentence in the README note you added.

**Comment by jponge** [comment]
Sounds like a very good start :-)

File: README.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ NOTE: If you encounter issues with your application still being running in the b
 
 The plugin provides a `vertxDebug` task enabling to debug your code.
 
-NOTE: Reloading is disabled while debugging.
+NOTE: Reloading is disabled while debugging. Moreover in order to prevent warnings while in debug mode, Vert.x options maxEventLoopExecuteTime and maxWorkerExecuteTime are set to `java.lang.Long.MAX_VALUE`
```
Comment (jponge) [inline README.adoc]: Not to be pedantic, but `maxEventLoopExecuteTime` and `maxWorkerExecuteTime` shall be between back-ticks to be rendered with a monospace font :-)

**Comment by tiaped** [comment]
:) You are right. Thanks for the comment.

File: src/main/kotlin/io/vertx/gradle/VertxPlugin.kt [hunk 3]
```diff
@@ -177,5 +179,48 @@ class VertxPlugin : Plugin<Project> {
...
+    val disableWorkerchecker = "-Dvertx.options.maxWorkerExecuteTime=" + java.lang.Long.MAX_VALUE
```
Comment (jponge) [inline src/main/kotlin/io/vertx/gradle/VertxPlugin.kt]: Did you know about Kotlin string expansion? You could rewrite as:

```kotlin
val disableWorkerchecker = "-Dvertx.options.maxWorkerExecuteTime=${java.lang.Long.MAX_VALUE}"
```

(same for some strings above)

Comment (tiaped) [reply src/main/kotlin/io/vertx/gradle/VertxPlugin.kt]: I'll have a look tomorrow.
Some more things to learn.
And I believe I saw a wrong debug at the end of the previous method.

**Pull request merged**